swift-sdk/IterableAPI.swift

```diff
@@ -114,6 +114,34 @@ import UIKit
         internalImplementation?.removeDeviceAttribute(name: name)
     }
     
+    /// Get the email address of the user of the SDK instance
+    public static func getEmail() -> String? {
+        return internalImplementation?.getEmail()
+    }
+    
+    /// Set the user of the SDK instance to the email address specified
+    ///
+    /// - parameter email: the email of the user for the SDK instance
+    /// - parameter token: the associated authentication token for the user
+    @objc(setEmail:token:)
+    public static func setEmail(_ email: String, _ token: String? = nil) {
+        internalImplementation?.setEmail(email, token)
+    }
+    
```

Comment:
We also need `setEmail:` method. We can't break any public API. Objc code that does not have token will not work with this. Swift will continue to work because token is optional parameter.

Reply:
Or do we already have the old method? In that case you can ignore this comment..

Reply:
You're correct, we don't have just `setEmail`/`setUserId`, I'll add that.

Reply:
I was mistaken, those functions are already generated on the Objective-C side, so we're fine!